build/package/gen-ssdlc-report.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+# Copyright 2025 MongoDB Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+set -eu
+
+release_date=${DATE:-$(date -u '+%Y-%m-%d')}
+
+export DATE="${release_date}"
+VERSION=""
+VERSION=$(git tag --list 'atlascli/v*' --sort=-taggerdate | head -1 | cut -d 'v' -f 2)
+export VERSION
+export AUTHOR="${AUTHOR:-$(git config user.name)}"
```

Comment:
Should the author be MongoDB? I think that `user.name` is the [apix-bot](https://github.com/apps/apix-bot) nowdays

Reply:
It's ok if it's the bot, the docs say "person or automation"